### DISCUSSION TITLE
右手の方が起動しない。

### DISCUSSION THREAD

**Discussion by hanjuku1** [Yowkees/keyball#428] [Q&A]
KEYBALL二台目の製作です。

症状は右手側のキーボードにおいて、　USB接続しても全くPC側がキーボードを認識しないという状況です。

ビルドガイドの３－８LEDまで半田付けを行いました。
左は正常だったので、右手側は確認せずLEDまで一部ハンダしてしまいましたが、認識しないことに気付きました。
以前教えていただいた通り、LEDの問題を切り分けるために、LEDCUTの部分を切断し絶縁してLEDを切り離しましたが、認識しませんでした。
キーソケットまでで何か失敗するようです。

以前購入時のダイオードが余っていたので、今回使用したのですが、今回と前回とでダイオードは見た目が異なります。まぜたとこで認識しないのでしょうか？
ただ単純にダイオードの向きが間違っているから認識しないのでしょうか？

PCが認識しない可能性として、何が間違っているか、お教えいただければ幸いです。

**Comment by hanjuku1** [comment]
すいません、自己解決できました。
肉眼でかなり発見しづらい場所で短絡していました。

ありがとうございました。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

ご自身で問題箇所を見つけていただきありがとうございました。
２台目ということで、お選びいただき誠にありがとうございます。

ぜひ末永くご愛用いただければ幸いです。
